dialog/src/main/java/net/md_5/bungee/api/dialog/body/DialogBody.java

```diff
@@ -2,12 +2,14 @@
 
 import lombok.Data;
 import lombok.NonNull;
+import lombok.experimental.Accessors;
 import org.jetbrains.annotations.ApiStatus;
 
 /**
  * Represents the body content of a {@link net.md_5.bungee.api.dialog.Dialog}.
  */
 @Data
+@Accessors(fluent = true)
```

Comment:
This can stay too - I think type() is treated separately and not as an API method